README.md

```diff
@@ -327,6 +325,29 @@ const draft = nylas.drafts.build({
   replyToMessageId: MESSAGE_ID,
 });
 
+// Enabling tracking
+
+  const draft = nylas.drafts.build({
+    "tracking":{
```

Comment:
let's update this to show how we would pass this tracking obj into `send`, instead of passing it into build